cysync/src/store/provider/syncProvider/index.tsx

```diff
@@ -1004,34 +1009,51 @@ export const SyncProvider: React.FC = ({ children }) => {
     setupInitial();
 
     // Refresh after 60 mins
-    if (intervals.current.length === 0) {
+    if (
+      intervals.current.length === 0 &&
+      process.env.IS_PRODUCTION === 'true'
+    ) {
       intervals.current.push(
         setInterval(async () => {
-          if (connected && process.env.IS_PRODUCTION === 'true') {
-            logger.info('Sync: Refresh triggered');
-            try {
-              addPriceRefresh({ isRefresh: true, module: 'refresh' });
-              await notifications.getLatest();
-              await transactionDb.failExpiredTxn();
-            } catch (error) {
-              logger.error('Sync: Error in refresh');
-              logger.error(error);
-            }
+          logger.info('Sync: Refresh triggered');
+          if (connectedRef) {
+            // Needs revamp
+            addPriceRefresh({ isRefresh: true, module: 'refresh' })
+              .then(() => {
+                logger.info('Sync: Price Refresh completed');
+              })
+              .catch(err => {
+                logger.error('Sync: Price Refresh failed', err);
+              });
```

Comment:
Add Price refresh irrespective of the connected status. Connection & Duplicates will be internally checked by the sync queue.

Reply:
pls check again